markdown/src/main/java/it/niedermann/android/markdown/MarkdownUtil.java

```diff
@@ -50,6 +50,7 @@
     private static final Pattern PATTERN_ORDERED_LIST_ITEM = Pattern.compile("^(\\d+).\\s.+$");
     private static final Pattern PATTERN_ORDERED_LIST_ITEM_EMPTY = Pattern.compile("^(\\d+).\\s$");
     private static final Pattern PATTERN_MARKDOWN_LINK = Pattern.compile("\\[(.+)?]\\(([^ ]+?)?( \"(.+)\")?\\)");
+    private static final Pattern PATTERN_MARKDOWN_TITLE = Pattern.compile("^# ");
```

Comment:
I am not that good in Regex, but this looks as it would just handle *exactly one* `#` character, right?
It would be a great opportunity to make it work for 1 - 6 `#` characters, so, replace:
1. `# `
2. `## `
3. `### `
4. `#### `
5. `##### `
6. `###### `

Reply:
Btw. there is already a regex `PATTERN_HEADINGS` that should do the same (but better). (Why) does this not work anymore?